server/events/comment_parser_test.go

```diff
@@ -56,7 +56,25 @@ func TestParse_HelpResponse(t *testing.T) {
 	}
 	for _, c := range helpComments {
 		r := commentParser.Parse(c, models.Github)
-		Equals(t, events.HelpComment, r.CommentResponse)
+		Equals(t, commentParser.HelpComment(false), r.CommentResponse)
+	}
+}
+
+func TestParse_HelpResponseWithApplyDisabled(t *testing.T) {
+	helpComments := []string{
+		"run",
+		"atlantis",
+		"@github-user",
+		"atlantis help",
+		"atlantis --help",
+		"atlantis -h",
+		"atlantis help something else",
+		"atlantis help plan",
+	}
+	for _, c := range helpComments {
+		commentParser.ApplyDisabled = true
+		r := commentParser.Parse(c, models.Github)
+		Equals(t, commentParser.HelpComment(true), r.CommentResponse)
```

Comment:
Isn't this just testing itself?

Reply:
Maybe, I modeled this test on the the test above TestParse_HelpResponse maybe its not needed?

Reply:
Ah okay. Well then we can blame the previous programmer (me 😄 ). We can leave it as is then. Must have been testing that we output help given these comments, not the actual contents of help which is tested elsewhere.